.github/workflows/test-e2e.yml

```diff
@@ -186,7 +186,7 @@ jobs:
             "custom-roles",
             "teams",
             "backup-config",
-            "data-federation",
+            "datafederation",
```

Comment:
👌 
But I suspect this change will uncover also testing errors, so waiting for those fixes.

Reply:
add fix commit on top